.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,72 @@
+name: Dependencies
+
+on:
+  schedule:
+    # View https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
+    - cron: '0 12 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Check development branches of dependencies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8.3, 3.9]
+        redis-version: [3, 4, 5, 6]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Start Redis
+      uses: supercharge/redis-github-action@1.1.0
+      with:
+        redis-version: ${{ matrix.redis-version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install git+https://github.com/andymccurdy/redis-py
+        pip install git+https://github.com/pallets/click
+        pip install -r dev-requirements.txt
+        pip install -e .
+
+    - name: Test with pytest
+      run: RUN_SLOW_TESTS_TOO=1 pytest --durations=5 > log.txt 2>&1
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: dependencies-error
+        path: log.txt
+      if: failure()
+
+  issue:
+    name: Create failure issue
+    runs-on: ubuntu-latest
+
+    if: failure()
+    needs: build
+
+    steps:
+    - uses: actions/download-artifact@v2
+      with:
+        name: dependencies-error
+        path: .
+
+    - name: Create failure issue
+      run: |
+        if [[ "$(curl --url https://api.github.com/repos/rpkak/rq/issues --request GET --data '{"creator": "github-actions"}')" != *"\""* ]]
```

Comment:
> Does this mean if a future build fails, it will create a new issue every day?

If will create only an issue if there isn't already an issue for that.

> Can we also run this only if CI on master passes? There have been a few instances where CI on master fails, if this is the case, then it shouldn't need to create a new issue.

Ok I can implement that.